doc/source/debug.rst

```diff
@@ -0,0 +1,188 @@
+    proxy-deployment-1227971824-mn2wd   1/1       Running        0          5h
+
+Here we can see the two JupyterHub pods, as well as a single user pod. Note
+that all user pods will begin with ``jupyter-``.
+
+In particular, keep an eye on the ``STATUS`` column. If a given
+pod contains something other than ``Running``, then something may be wrong.
+
+In this case, we can see that our user's pod is in the ``ErrImagePull`` state.
+This generally means that there's something wrong with the Docker image that
+is defined in ``singleuser`` in our helm chart. Let's dig further...
+
+``kubectl describe pod``
+~~~~~~~~~~~~~~~~~~~~~~~~
+To see more detail about the state of a specific pod, use the following
+command::
+
+    kubectl describe pod <POD_NAME>
+
+This will output several pieces of information, including configuration and
+settings for the pod. The final section you'll see is a list of recent
+events. These can be particularly informative, as often an error will
+show up in this section.
+
+**Real world scenario:** In our case, one of the lines in the events page
+displays an error::
+
+      $ kubectl describe pod jupyter-choldgraf --namespace=jhub
+      ...
+      2m            52s             4       kubelet, gke-jhubtest-default-pool-52c36683-jv6r        spec.containers{notebook}       Warning         Failed           Failed to pull image "jupyter/scipy-notebook:v0.4": rpc error: code = 2 desc = Error response from daemon: {"message":"manifest for jupyter/scipy-notebook:v0.4 not found"}
+      ...
+
+It seems there is indeed something wrong with the Docker image. Let's confirm
+this by getting another view on the events that have transpired in the pod.
+
+``kubectl logs``
+~~~~~~~~~~~~~~~~
+If you only want to see the latest logs for a pod, use the following command::
+
+    kubectl logs <POD_NAME>
+
+This will output a list of the recent events for the pod. Parse these logs
+to see if something is generating an error.
+
+**Real world scenario:** In our case, we get this line back::
+
+    $ kubectl logs jupyter-choldgraf --namespace=jhub
+    Error from server (BadRequest): container "notebook" in pod "jupyter-choldgraf" is waiting to start: trying and failing to pull image
+
+Now we are sure that something is wrong with our Docker file. Let's check
+our ``config.yaml`` file for the section where we specify the user's
+Docker image. Here we see our problem::
+
+  singleuser:
+  image:
+      name:
+          jupyter/scipy-notebook
+
+We haven't specified a ``tag`` for our Docker image! This is required in
+JupyterHub and not specifying a tag will cause the image pull to fail.
+
+To fix this, let's add a tag to our ``config.yaml`` file::
+
+  singleuser:
+  image:
+      name:
+          jupyter/scipy-notebook
+      tag:
+          ae885c0a6226
+
+and run a helm upgrade::
+
+    helm upgrade jhub jupyterhub/jupyterhub --version=v0.4 -f config.yaml
+
+.. note::

```

Comment:
Instead of a note (folks are more likely to read the sentence while executing and the note only if the command fails - let's help them get it correct on the first pass), let's continue the sentence below the command. 

where `jhub` is the Kubernetes namespace (substitute the namespace that you chose during setup).